testcases/multizone_residential_hydronic/doc/MultiZoneResidentialHydronic.html

```diff
@@ -0,0 +1,658 @@
+<html>
+<p>
+This is the multi zone residential hydronic emulator model
+of BOPTEST.
+</p>
+<h3>Building Design and Use</h3>
+<h4>Architecture</h4>
+<p>
+The emulator building model represents a residential French
+dwelling compliant with the French Thermal regulation of 2012,
+i.e. the French national building energy regulation. Therefore,
+the typology is defined to be representative of French new
+dwellings. Its area is approximately 120 m&sup2; (<i>S_ref</i>).
+The following figure shows the building layout and
+a sketch of the hydraulic system. The coloured elements in the scheme
+represent the controllable components
+through the BOPTEST interface. The dimensions are provided in metres.
+
+</p>
+<p align=\"center\">
+<img alt=\"Simulated residential dwelling\"
+src="images/layout.png\">
```

Comment:
@dhblum I can't see the image unless:
```suggestion
src="images/layout.png">
```
Is it the same for you?

Reply:
It works either way for me in Firefox on Ubuntu 18.04.  Let's take your suggestion since it seems to work better on your setup.